src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -170,16 +188,26 @@
     private String gerritVerifiedCmdBuildStarted;
     private String gerritVerifiedCmdBuildNotBuilt;
     private String gerritFrontEndUrl;
-    private Integer gerritBuildStartedVerifiedValue = null;
-    private Integer gerritBuildSuccessfulVerifiedValue = null;
-    private Integer gerritBuildFailedVerifiedValue = null;
-    private Integer gerritBuildUnstableVerifiedValue = null;
-    private Integer gerritBuildNotBuiltVerifiedValue = null;
-    private Integer gerritBuildStartedCodeReviewValue = null;
-    private Integer gerritBuildSuccessfulCodeReviewValue = null;
-    private Integer gerritBuildFailedCodeReviewValue = null;
-    private Integer gerritBuildUnstableCodeReviewValue = null;
-    private Integer gerritBuildNotBuiltCodeReviewValue = null;
+    @Deprecated
+    public Integer gerritBuildStartedVerifiedValue = null;
```

Comment:
why make these `public` if they are deprecated?

Reply:
I don't quite recall why I made those deprecated since they need to be used, so I removed the 'Deprecated' annotations